src/components/controls/annotatedTitle.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { FaInfoCircle } from "react-icons/fa";
+import {StyledTooltip, HeaderIconContainer, HeaderTitle} from "./styles";
+
+type Props = {
+  /** Title to display for the control. */
+  title: string
+
+  /** Informational tooltip element to display on hover. */
+  tooltip?: JSX.Element
+
+  /** Indicates mobile display. */
+  mobile: boolean
+}
+
+/**
+ * A title and tooltip to be shown in a control header.
+ * The tooltip is not shown on mobile.
+ */
+export const AnnotatedTitle = ({title, tooltip=undefined, mobile}: Props) => {
+  return (
+    <span>
+      <HeaderTitle>{title}</HeaderTitle>
+      {tooltip && !mobile && (
+        <>
+          <HeaderIconContainer data-tip data-for={title}>
+            <FaInfoCircle/>
+          </HeaderIconContainer>
+          <StyledTooltip place="bottom" type="dark" effect="solid" id={title}>
```

Comment:
The size of the info-icon has become much smaller in this PR - I assume this wasn't on purpose? I think the larger size is preferable

Reply:
Oops, this wasn't on purpose. Thanks for catching!

Reply:
Fixing with da91b15b7c25b705b86fca80917df4ef2e7cb8ae